src/EditorFeatures/TestUtilities/Workspaces/CSharpTestWorkspaceFixture.cs

```diff
@@ -9,7 +9,7 @@ public class CSharpTestWorkspaceFixture : TestWorkspaceFixture
     {
         protected override TestWorkspace CreateWorkspace()
         {
-            return TestWorkspace.CreateCSharp(
+            return TestWorkspace.CreateCSharp2(
```

Comment:
Because we have 2 overloads with optional parameters that are ambiguous when called only with some parameters. These helpers really needs some cleanup. But that's beyond the scope of this PR. I'd file an issue, but we would ignore that issue anyways.

---
In reply to: [162472376](https://github.com/dotnet/roslyn/pull/24316#discussion_r162472376) [](ancestors = 162472376)